flang/test/Lower/OpenMP/Todo/omp-default-clause-inner-loop.f90

```diff
@@ -8,7 +8,7 @@
 ! CHECK: omp.parallel   {
 ! EXPECTED: %[[PRIVATE_Y:.*]] = fir.alloca i32 {bindc_name = "y", pinned, uniq_name = "_QFEy"}
 ! EXPECTED: %[[PRIVATE_Z:.*]] = fir.alloca i32 {bindc_name = "z", pinned, uniq_name = "_QFEz"}
-! CHECK: %[[TEMP:.*]] = fir.alloca i32 {adapt.valuebyref, pinned}
+! CHECK: %[[TEMP:.*]] = fir.alloca i32 {bindc_name = "x"
```

Comment:
Nit: if it's not too much work, it would be nice to keep braces balanced, using something like:
`{bindc_name = "x", {{.*}}}`

Reply:
Done.